`microsoft` 英語を日本語にそろえる (#1188)
utelecon/utelecon.github.io: src/pages/en/microsoft/signin.md

---
title: Sign in to Microsoft systems using UTokyo Account
breadcrumb:
  title: Sign in with UTokyo Account
redirect_from:
  - /en/faq/msaccount-troubleshooting
---

## About
{:#about}

This page explains how to sign in to Microsoft systems using UTokyo Account.

The University of Tokyo provides [UTokyo Microsoft License](./), which allows you to use Microsoft systems with UTokyo Account. If you are prompted to sign in to Microsoft systems using UTokyo Account, please follow the instructions in "[Sign in to Microsoft systems](#instruction)" below.

> If you are already signed in with a different Microsoft account and want to switch to UTokyo Account, please refer to "[Switching to UTokyo Account when signed in with a different Microsoft account](#switch_accounts)". In addition, the recommended method for using multiple Microsoft accounts in the same browser is explained in "[Using multiple Microsoft accounts in the same browser](#multiple_accounts)". Please refer to this section if you have two UTokyo Accounts, one for students and one for faculty and staff, or if you want to use your personal or other organization's Microsoft account in addition to UTokyo Account.

Reviewer: めちゃめちゃ細かくて正直どっちでもいいのですが、".と." のどっちがいいんですかね？/installのページとこちらで違っていたので気になりました。uteleconとしてどちらにそろえるべきなどあれば

For general information about UTokyo Microsoft License and details of available systems, please refer to "[UTokyo Microsoft License](./)".

## Sign in to Microsoft systems
{:#instruction}

This section explains how to sign in using your UTokyo Account if a screen appears requesting you to sign in to the Microsoft system.

When signing in to Microsoft systems using UTokyo Account, contrary to signing in to other systems, you may encounter a sign-in screen that includes the Microsoft logo. In that case, please follow the steps below to sign in. On the other hand, if the screen is not as shown below but a normal sign-in screen (with a picture of Yasuda Auditorium) is displayed, please sign in following "[Sign in to the ICT systems with your UTokyo Account](/en/utokyo_account/#signin)".

<figure class="gallery">
<img src="img/microsoft-signin-instruction-1.png" class="medium center border">

> <img src="img/microsoft-signin-instruction-3.png" class="medium center border">

Reviewer: これって英語版はありますかね？

Reviewer: 自分も探してみても見つからず、、安田講堂画面はもう出ないんでしたっけ？

Author: 赤門画面になっているらしいので、これで良いのですかね……？

<img width="300" alt="image" src="https://github.com/user-attachments/assets/152968ef-ed03-4f7a-a157-7aed60833640">

Reviewer: ありがとうございます、いいと思います。

2. After the message "Taking you to your organization's sign-in page" is displayed, you will be automatically redirected to the sign-in page with the image of Yasuda Auditorium. Follow the instructions in "[Sign in to the ICT systems with your UTokyo Account](/utokyo_account/#signin)" to sign in.
    <details>
    <summary>Help when this procedure does not work</summary>

    - If you see **"Enter a valid email address, phone number, or Skype name."** or **"We couldn't find an account with that username. Try another, or get a new Microsoft account."**
        - The input is incorrect. Please check if you have entered "10-digit common ID + `@utac.u-tokyo.ac.jp`" correctly.
    - If you see **"That Microsoft account doesn't exist. Enter a different account or get a new one."**
        - You cannot sign in with UTokyo Account on this sign-in page. If you see this screen even though the system supports signing in with UTokyo Account, please contact the [support desk](/en/support/).
    </details>

> ## You are signed in with other Microsoft accounts and switch to using your UTokyo Account

Reviewer: ```suggestion
## Switch to UTokyo Account if you are signed in with another Microsoft account
```

Author: 他のところと合わせて ~ing 形にしつつ直しました

Instead of using a different browser for each Microsoft account to avoid this issue, you can use multiple Microsoft accounts in the same browser by using the browser's incognito browsing function or using multiple profiles.

### Using incognito browsing
{:#secret_browsing}

If you want to temporarily use a different Microsoft account, it is convenient to use the incognito browsing function of your browser.

Incognito browsing is a function that allows you to use the browser without having the browser remember information such as browsing history or sign-in information. While using the incognito browsing function, not only is no information remembered, but the browser is also unaffected by information that is remembered when you normally use the browser. In other words, in incognito browsing, you can view web pages without being affected by the sign-in information stored in the browser. Therefore, if you sign in to a Microsoft account using the incognito browsing function, you can use the Microsoft system without being affected by the other Microsoft accounts you normally sign in to.

> Note that the incognito browsing method is only suitable if you want to temporarily sign in with a different account. This is because the information of the signed-in account is not retained even if you close the browser window. If you use a Microsoft account frequently and want to use it continuously, the "[Use multiple profiles](#profiles)" method described below is convenient.

Reviewer: 日本語もそうなのですが、「ブラウザーを閉じても」サインインしたアカウントの情報が保持されないのところが正直よくわからず、、

Author: 日本語の係り受けが不自然な気がしています。
「ブラウザのウィンドウを閉じても〜」のところを「ブラウザのウィンドウを閉じると消えてしまう」という言い方にするのが良いかなと思っています。

Reviewer: そのように修正していただけるといいと思います。

Author: https://github.com/utelecon/utelecon.github.io/pull/1188/commits/b5f1ddad2b73e7b70e640cc3e8b56ff168131546
修正しました
